tests/manage/pv_services/test_delete_provisioner_pod_while_thick_provisioning.py

```diff
@@ -1,143 +0,0 @@
-import logging
-from concurrent.futures import ThreadPoolExecutor
-import pytest
-
-from ocs_ci.framework.testlib import (
-    ManageTest,
-    tier4,
-    tier4a,
-    polarion_id,
-    bugzilla,
-)
-from ocs_ci.helpers.helpers import (
-    verify_volume_deleted_in_backend,
-    default_thick_storage_class,
-    check_rbd_image_used_size,
-    default_ceph_block_pool,
-)
-from ocs_ci.ocs import constants
-from ocs_ci.ocs.resources.pod import get_fio_rw_iops
-from ocs_ci.ocs.resources.pvc import get_all_pvcs
-from ocs_ci.helpers import helpers, disruption_helpers
-
-
-logger = logging.getLogger(__name__)
-DISRUPTION_OPS = disruption_helpers.Disruptions()
-
-
-@tier4
```

Comment:
Should we remove the tier marker from the test instead of deleting the entire test file?
If there is support for this feature in the future, we can simply enable the test back

Reply:
The test cases are currently inactive. I don't think this feature will be enabled in future due to the difficulty in achieving it and  there were another methods mentioned in the Jira which can replace this feature.